plugins/admin.py

```diff
@@ -0,0 +1,131 @@
+"""admin actions
+
+Commands:
+!kick
+!ban
+!unban
+
+# ../config.py
+# int | str | list
+ADMIN_CHATS = -1234567890123
+"""
+import asyncio
+from time import time
+from pyrogram import Client, filters
+from pyrogram.types import Message
+from pyrogram.errors import UserNotParticipant
+from config import ADMIN_CHATS
+
+USAGE_KICK = """**Usage**:
+*admin reply to a user message*
```

Comment:
the grammar a bit messed up here, i would write

admin should reply to a user message with `!ban` [reason]

Reply:
`*someting*`
this may be a common way to describe a behavior (usually something was happening), such like `*sigh*`, similar to `/me doing something`. or maybe I misunderstood that (could be just assume a bold/italic text)

anyway, I'll probably remove the usage text and use custom filters to just allow some commands to only allow the action when some condition meets. No error or any info will be given when it does not meet the condition.
Because I think the code would be a little bit simpler that way.

Reply:
i have no problem with the styling, was just talking about the text